Add a metaTitle and a metaDescription to every doc pages (#233)
strapi/documentation: docs/user-docs/latest/users-roles-permissions/managing-administrators.md

---
title: Managing Administrators - Strapi User Guide

> description: Administrators are the users of an admin panel of a Strapi project. They can be managed with the Role-Based Access Control (RBAC) feature.

Reviewer: ```suggestion
description: Instructions to manage the administrators of a Strapi application with the Role-Based Access Control (RBAC) feature.
```
